### ISSUE TITLE
Implement Floating submit button

### ISSUE THREAD

**Issue by bdougie** [open-sauced/hot#132]

**Comment by nulfrost** [comment]
I can take this on, since we are going to be using radix we can use the [popover](https://www.radix-ui.com/docs/primitives/components/popover) component to achieve this easily

**Comment by nulfrost** [comment]
.take

**Comment by bdougie** [comment]
Hey @nulfrost. Checking in to see if you still want to tackle this. We are using the beta branch for these UI features.

**Comment by nulfrost** [comment]
Yeah I can take a look at this tonight, was mostly waiting for the radix stuff to be merged in but I can see that's been wrapped up

**Comment by bdougie** [comment]
Checking in to see if you still have time for this.

**Comment by 0-vortex** [comment]
> Haven't had a chance to work on this recently, I think the two parts I got stuck on were making the button float on the page and then making sure the submit box was always positioned above the button no matter where you scrolled on the page.
> 
> For the first problem there seems to be some height issue where the content on the page is overflowing the parent height so the button is placed like halfway down the page. Tried messing around with it so the height grows with the content but couldn't get it working.
> 
> For the second problem, it's really just getting the position of the button on the page and then making sure the submit box is placed above it, I think the approach I was going with initially was wrong and it can all be done in CSS. Haven't found any similar solutions that implement this so if you know of any let me know.

I can take a look at this but it's likely the current layout was pretty static - a responsive flex approach would be desired however I am not sure if that would imply messing with some of the other components as well. Nonetheless, a CSS approach would be best as calculating screen values and relying on client screen dimensions triples the work long term due to:
- desktop layout
- responsive layout
- server-side rendering

**Comment by shamimbinnur** [comment]
@bdougie is the backend code implemented in any branch? If it is implemented, I am gonna start working from that branch.

**Comment by shamimbinnur** [comment]
Floating repo submission UI implemented
#179
